src/khoj/routers/helpers.py

```diff
@@ -927,11 +928,13 @@ async def text_to_image(
                 response = state.openai_client.images.generate(
                     prompt=improved_image_prompt,
                     model=text2image_model,
+                    style="vivid",
```

Comment:
Maybe use an all caps magic variable name to set this, since it contains specific business logic? e.g., `OPENAI_IMAGE_GEN_STYLE = "vivid"`. Generally prefer to have things like this called out in a way where it's obvious that this variable is non-standard and where to learn more.